src/controllers/routes.js

```diff
@@ -1,10 +1,13 @@
 const express = require('express');
-
 const router = express.Router();
-
+const signup = require('./signup');
 const login = require('./login');
 
-router.get('/login', login.get);
```

Comment:
why you deleted this line? you  will make conflict

Reply:
i replaced it on the next lines